src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static String getDataFromFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder fileInfoLineBuilder = new StringBuilder();
+            String lineFromFile = reader.readLine();
+            while (lineFromFile != null) {
+                fileInfoLineBuilder.append(lineFromFile).append(System.lineSeparator());
+                lineFromFile = reader.readLine();
+            }
+            return fileInfoLineBuilder.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find File", e);
+        }
+    }
 
+    private static String getReportFromFile(String fromFileName) {
+        int totalSupply = 0;
+        int totalBuy = 0;
+        String[] fileData = getDataFromFile(fromFileName).split(System.lineSeparator());
+        for (String data : fileData) {
+            if (data.indexOf('b') == 0) {
+                totalBuy = totalBuy + Integer.parseInt(data.replaceAll("[^0-9]", ""));
+            } else {
+                totalSupply = totalSupply + Integer.parseInt(data.replaceAll("[^0-9]", ""));
```

Comment:
let's make variable for `Integer.parseInt(data.replaceAll("[^0-9]", ""))` one time and use it on both lines